### ISSUE TITLE
If getting 'str has no attribute decode'

### ISSUE THREAD

**Issue by Shantanugupta1118** [Shantanugupta1118/Social-Distancing-and-Face-Mask-Detection#5]
### **SOLUTION APPROACHES TO RESOLVE THE OCCURED ERRORS.**

try the following steps to resolve the error. one by one

* keep updating all libraries
* should this project use in one tensorflow environment only
* uninstall tensorflow, keras, tensorflow-gpu by using 
              - `conda uninstall tensorflow`
              - `conda uninstall tensorflow-gpu`
              - `conda uninstall keras`
               ,respectively

and install it again using **install** instead of uninstall same.
               - `conda install tensorflow`
              - `conda install tensorflow-gpu`
              - `conda install keras`
after run main.py in the file directory using `python main.py`

Thank you.
if it doesn't work then REPLY below
